syft/workers/virtual.py

```diff
@@ -1,9 +1,17 @@
+from time import sleep
+from typing import Union
```

Comment:
Is there actually a Union type in this file?

Reply:
Fixed!